docs/6.x/cluster.md

```diff
@@ -208,6 +208,54 @@ $ curl -sk https://localhost:7575 | python -m json.tool
 
 In this case the response HTTP status code will be `503 Service Unavailable`.
 
+### Cluster Status History
+
+Running `gravity status history` will display a history of changes to the 
+cluster status. 
+
+Example output may looks something like the following:
+
+```bsh
+$ gravity status history
+Feb  6 21:36:07 UTC [Node Degraded]     node=node-1
+Feb  6 21:36:11 UTC [Node Degraded]     node=node-2
+Feb  6 21:36:25 UTC [Node Degraded]     node=node-3
+Feb  6 21:36:56 UTC [Probe Succeeded]   node=node-1 checker=node-status
+Feb  6 21:36:58 UTC [Probe Succeeded]   node=node-2 checker=node-status
+Feb  6 21:36:58 UTC [Probe Succeeded]   node=node-2 checker=time-drift
+Feb  6 21:36:59 UTC [Probe Succeeded]   node=node-3 checker=node-status
+Feb  6 21:37:07 UTC [Probe Succeeded]   node=node-1 checker=kube-apiserver
+Feb  6 21:37:07 UTC [Node Recovered]    node=node-1
+Feb  6 21:37:08 UTC [Probe Succeeded]   node=node-2 checker=kube-apiserver
+Feb  6 21:37:08 UTC [Node Recovered]    node=node-2
+Feb  6 21:37:11 UTC [Probe Succeeded]   node=node-3 checker=kube-apiserver
+Feb  6 21:37:11 UTC [Node Recovered]    node=node-3
+```
+
+Here's an example of how to view the history remotely via `tsh`:
+
+```bsh
+$ tsh --cluster=production ssh admin@node gravity status history
+```
+
+The `gravity status history` command is an additional tool to help debug issues
+with a Cluster. The `gravity status` command only displays the current status
+and provides limited visibility into the state of the Cluster. The
+`gravity status history` command is there to help fill in the gaps. The history
+lets you observe when and where problems have occurred within the Cluster.
+
+There are just a few event types that are currently being tracked.
+- `Node Degraded` / `Node Recovered` specifies a change in the node status. The node
+key specifies the name of the node (node-1, node-2, node-3).
+- `Probe Succeeded` / `Probe Failed` specifies a change in a probe result. The checker
+key specifies the name of the health check (time-drift, kube-apiserver).
+- A few other events will be tracked and are WIP. We plan to track cluster 
+resizing, gravity upgrades, network partitions, etc...
+- If you have suggestions for other types of events to be tracked, let us know [here](https://github.com/gravitational/gravity/issues/new/choose).
```

Comment:
I don't think I would document statements like this, or that this tool is WIP above. I don't think we should be publishing docs that say what the product might do, just what it's capable of.